regions_www/tabs/region_stats/commute/lsoa/wales/region_stats.html

```diff
@@ -46,11 +46,11 @@
 
 <p><br></p>
 
-<h3>The  Wales region</h3>
+<h3>Wales</h3>
 
 <p>Figure 1 shows a map of  Wales, including the regional boundary (thick black border), local authority boundaries (blue lines) and LSOA zones in the region (grey with a black border). The straight green lines represent the top 100 flows in terms of the number of commuters by any mode.</p>
 
-<p><em>Figure 1: Map of the  Wales region</em>
+<p><em>Figure 1: Map of Wales</em>
```

Comment:
Thanks for adding these important changes - people from Wales would understandably want to be seen as more than a 'region'.